code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -88,14 +88,24 @@
 
 /obj/item/weapon/tank/jetpack/void
 	name = "Void Jetpack (Oxygen)"
-	desc = "It works well in a void."
+	desc = "A simple retro jetpack. This one is painted orange."

```

Comment:
Why change this joke description?

Reply:
I thought that a more descriptive description might be helpful as there are now three versions.

Reply:
It's not a very well joke description though

Reply:
Don't change this, please.